docs/New Game.md

```diff
@@ -31,14 +31,85 @@ game to have proper pickups:
+applied to the game. 
 
-In order to export a game, a patcher is required. Typically, the patcher will be included in Randovania for seamless exports. This is a requirement for non-experimental games. Experimental games, however, may prefer an alternative implementation: refer to Corruption as an example of exporting data to be passed to a standalone patcher.
+In most cases, this involves readings the files from a user-provided copy of the game and creating either
+a new ROM/ISO file or a mod file.
 
-**TODO**: details
+This process is split into two parts:
 
+### Patch Data Factory
+
+Each game defines a class that inherits from `PatchDataFactory`. Given a `LayoutDescription`, a `CosmeticPatches` and 
+`PlayersConfiguration`, it creates a JSON Object that will be passed as an argument to the Game Exporter.
+
+### Game Exporter
+
+Each game also defines a class that inherits from `GameExporter`. Given the JSON created by the `PatchDataFactory` and a
+`GameExportParams`, provided by your game's `GameExportDialog`, it's expected to create all the necessary files for the
+user to play the game.
+
+Since Randovania does not modify the game files, a separate Python package must be created which is responsible for
+these changes. This package is called the patcher for the game.
+
+You are free to implement this package however you want, but we recommend the following:
+
+#### Input Data
+
+The arguments for the patcher should be:
+- A json file, also called the patcher data
+- Paths to necessary game files (input path)
+- Paths to an output folder (output path)
+- A status update function
+
+Any other parameter should come from the json file.
+
+#### Language
+
+Best:
+- Python
+
+Good (Known to integrate without issues):
+- C
+- C++
+- Rust
+- Lua
+
+Likely Good (Should integrate well, never tested):
+- Languages that compile to static library, such as Go
+
+Ok (Expect additional work to integrate properly, avoid if possible):
+- C#
+
+Avoid languages that require the user to install a runtime, such as Java.
+
+#### Standalone Use
+
+Allow your patcher to be used directly in the command line, by passing the game files and a json file.
+
+This makes it easy to test the patcher without having to run it through Randovania, and helps better identify issues.
+
+#### Json Schema
+
+Define a [JSON Schema](https://json-schema.org/understanding-json-schema/) that defines the structure of the patcher data.
+This makes for a good documentation of what is the input for the patcher, as well as a very good safety against
+misconfiguration in Randovania.
 
 ### Multiworld
 
+Before a game can work in Multiworld, it must first work properly as a solo game. It is highly recommended to finish the basic integration first.
+
 **TODO**
+
+# Rules for new games
+
+Randovania has a high bar of quality for our game integrations, and we wish to keep it that way. In order to help
+with that goal, there are rules that need to be met before we can accept a game.
```

Comment:
Randovania has a high quality standard for game integration. In order for a new game to be added, the following rules must be adhered to: